spring-integration-smb/src/test/resources/log4j2-test.xml

```diff
@@ -7,7 +7,7 @@
 	</Appenders>
 	<Loggers>
 		<Logger name="org.springframework.integration" level="warn"/>
-		<Logger name="org.springframework.integration.smb" level="warn"/>
+		<Logger name="org.springframework.integration.smb" level="info"/>
```

Comment:
Why move to `info`?
Yes, I'm doing that locally as well when I test and debug something, but it is better to keep CI build as clean as possible.
And that also would be make it faster without extra logging messages.

Reply:
That was for the Main class to verify functionality against a real SMB file share... I tested this against a Dev file server on a Windows VM in AWS, and was able to debug it fully for all functionality that I couldn't debug using JUnit and a Mockito mock session.

This is a configuration file only for testing code in the test folder.

Reply:
sure! As I said: it's OK to have something changed for local testing or have something custom for debugging or confirmation, but that's not OK to pollute source code of the project with "dead" files or undesired changes.

So, I vote for removing that `Main` and its respective XML config and reverting this back to `warn`

Reply:
I don't consider it a dead file, it was very helpful to verify functionality so that it works as intended. I can see it being helpful to others, I can remove it and keep it to myself if you want.